test/jdk/java/lang/invoke/lambda/MetafactoryArgValidationTest.java

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
copyright year needs update.

Reply:
Fixed. And I bumped it on the other files, too.